### PR TITLE
ATTEND-811: Create v4.attendees.markdown and Swagger

### PR DESCRIPTION
**Pull request by t-reid-sap**

### CODE REVIEW

**Comment by jonikamelcher** [comment]
Hi @t-reid-sap, please let us know when this PR is ready for review. Once we merge it, you will not be able to make any more changes while it is in review by the writers. When the PR is merged, that does not mean the documentation is approved, it is simply more time effective for us to merge the PR and make manual edits. We will contact you if we need more information or let you know when it is approved and completed.

Please let us know when you are finished adding commits. Thank you.

**Comment by t-reid-sap** [comment]
> Hi @t-reid-sap, please let us know when this PR is ready for review. Once we merge it, you will not be able to make any more changes while it is in review by the writers. When the PR is merged, that does not mean the documentation is approved, it is simply more time effective for us to merge the PR and make manual edits. We will contact you if we need more information or let you know when it is approved and completed.
> 
> Please let us know when you are finished adding commits. Thank you.

@jonikamelcher, we are ready for official review
Please include this PR alongside consideration for public release: https://github.com/SAP-docs/preview.developer.concur.com/pull/813 (@chandra-nadukula)

**Comment by t-reid-sap** [comment]
Apologies as we make several more edits. This should be completed within a matter of minutes

**Comment by t-reid-sap** [comment]
@jonikamelcher, changes are now complete

File: src/api-reference/expense/attendees/v4.attendees.markdown [hunk 1]
```diff
@@ -0,0 +1,541 @@
+---
+title: Attendees v4
+layout: reference
+---
+
+{% include prerelease.html %}
```
Comment (chandra-nadukula) [inline src/api-reference/expense/attendees/v4.attendees.markdown:6]: should we be including this pre release tag?

Comment (jonikamelcher) [reply src/api-reference/expense/attendees/v4.attendees.markdown:6]: This will be removed by the tech writers right before release.

**Review by chandra-nadukula — APPROVED** [review]

**Review by jonikamelcher — APPROVED** [review]
:shipit:

**Pull request merged**